thumbor/optimizers/gifv.py

```diff
@@ -28,7 +28,7 @@ def optimize(self, buffer, input_file, output_file):
             command_params,
             output_file,
         )
-        os.system(command)
+        subprocess.call(command, shell=True)
```

Comment:
`subprocess.call(command, shell=True, stdin=open(os.devnull))`

ffmpeg somehow tries to read from stdin and messes up the shell.